ui/packages/consul-ui/app/components/topology-metrics/card/index.scss

```diff
@@ -74,5 +74,37 @@
   }
   .details {
     padding: 0 12px 12px 12px;
+    > *:not(:last-child) {
+      padding-bottom: 6px;
+    }
+    span {
+      display: none;
+    }
+    .group {
+      display: grid;
+      grid-template-columns: 20px 1fr;
+      grid-template-rows: repeat(2, 1fr);
+      grid-template-areas:
+        'partition partition'
+        'union namespace';
+      span {
+        display: inline-block;
+        grid-area: union;
+        padding-left: 7px;
+        margin-right: 0px;
+      }
+      span::before {
+        margin-right: 0px;
+        @extend %with-union-mask, %as-pseudo;
+        background-color: rgb(var(--tone-gray-500));
+      }
```

Comment:
I wondered, if this `span` is just for decorative purposes, could it use the pseudo element of the `dl` or `dd` here? Pretty sure thats what I ended up doing for the bucket list component. 

A bigger thing and not necessarily actionable here, but it would be nice if the same Bucket::List component could be used but just be styled differently depending on whether we want a slash separator or a `union` separator. The markup that Bucket::List uses is almost exactly the same as this. Anyway, not important this last bit (or the first bit for that matter), just half thinking out loud.